incubator/keycloak/templates/configmap.yaml

```diff
@@ -22,6 +22,12 @@ data:
   {{- end }}
 
     exec /opt/jboss/docker-entrypoint.sh -b 0.0.0.0 {{ .Values.keycloak.startParams }}{{- if $highAvailability }} --server-config standalone-ha.xml{{ end }}
+
+  {{- with .Values.keycloak.postStartScript }}
+    echo 'Running custom post-start script...'
```

Comment:
It's too bad this commit didn't make it in.  Having a post-start script is extremely valuable, as using the start params can only do so much.   

For example, because of https://issues.jboss.org/browse/KEYCLOAK-4923, importing clients with associated service account roles doesn't work so having a postStartScript really helps solving that problem.

Could be hooked to `lifecycle.postStart.exec`